### PR TITLE
feat($config): Allow override of badge colors

### PR DESCRIPTION
**Pull request by bb220**
**Summary**
Allow badge colors to be overridden via https://vuepress.vuejs.org/default-theme-config/#simple-css-override
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:
n/a
**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: lib/default-theme/styles/config.styl [hunk 1]
```diff
@@ -4,6 +4,9 @@ $textColor = #2c3e50
 $borderColor = #eaecef
 $codeBgColor = #282c34
 $arrowBgColor = #ccc
+$badgeGreen = #42b983
```
Comment (ulivz) [inline lib/default-theme/styles/config.styl]: Could "$badgeTip" be better?

Comment (bb220) [reply lib/default-theme/styles/config.styl]: Good call.  Updated :+1:

**Comment by ulivz** [comment]
Could you also help update the docs and create a PR for 1.x?

Related docs: 

- 0.x: https://vuepress.vuejs.org/default-theme-config/#simple-css-override
- 1.x: https://v1.vuepress.vuejs.org/config/#palette-styl

**Comment by bb220** [comment]
Yes, sure thing.  Thanks for the refs.

**Comment by f3ltron** [comment]
@bb220 i didnt check all the pr yet is the pr for v1 is created ? Is this pr is still available to be merged @ulivz  @bb220

**Comment by chryw** [comment]
Any update on this thread? This looks good to me. Could you merge? Thanks!

**Pull request merged**